dushimeemma/fs-simba-clockwork
# Revert "CWM-2 (feature): authentication (#1)"

Description

This PR introduces authentication on the app

Type of change 

[ ] Bug fix (Non breaking change that fixes an existing issue)

[x] New Feature (Non breaking change which adds funcitonality)

[ ] Breaking change (New feature that removes or causes an existing feature not to work as expected)

[ ] This change requires a documentation update

How Has This Been Tested?

 [x] Unit tests

[x] Component Tests

[x] Integration Tests

How Should This Be Manually tested:

- Clone this repo.

- Checkout the branch git checkout branch

- Run yarn install

- Run yarn dev

Checklist:

[x] My code follows all the styles and conventions outlined for this project

[x] I have linted my code before submission

[x] I have made my corresponding changes to the documentation

[x] My changes generates no new warning

[x] I have added tests that prove my feature is working or my fix is effective

[x] New and existing tests pass locally with my changes

JIRA

[CWM-2](https://www.notion.so/Authentication-a47f5a52cc9947feb7c83e5b892cf1db)
